app/controllers/coronavirus_form/check_answers_controller.rb

```diff
@@ -112,6 +116,20 @@ def product_details(products)
     end
   end
 
+  def transport_type(question)
+    answer = []
+    answer << session["transport_type"].flatten.to_sentence
```

Comment:
[to_sentence is cool](https://apidock.com/rails/Array/to_sentence). Hadn't seen that before!